app/scripts/controllers/client/ClientController.js

```diff
@@ -55,14 +55,13 @@
                     resourceFactory.globalSearch.search({query: searchString , resource: "clients,clientIdentifiers",exactMatch: exactMatch}, function (data) {
                         var arrayLength = data.length;
                         for (var i = 0; i < arrayLength; i++) {
-                            var result = data[i];
+                            var result = data[i];                            
```

Comment:
Unnecessary whitespaces. You should consider configuring your editor or git configuration to auto-trim trailing whitespaces.